fs/fspath/path_test.go

```diff
@@ -19,34 +19,38 @@ var (
 
 func TestCheckConfigName(t *testing.T) {
```

Comment:
You appear to have overwritten this test instead of making a new one?

Reply:
Yes, I extended the `TestCheckConfigName` test by simply adding a second "result" value and using the same input to test both `CheckConfigName` and `MakeConfigName` (should probably have renamed the test). This to avoid another "currently-identical-soon-just-almost-identical" copy of all the input strings - like there to some degree already is in the separate `TestCheckRemoteName`. But I could split it into a separate test, duplicating the same input, if you think that is better, that is completely fine as well..?